db/schema.rb

```diff
@@ -34,11 +34,11 @@
     t.datetime "created_at",     :null=>false
     t.datetime "updated_at",     :null=>false
     t.string   "app_key"
+    t.string   "app_secret"
     t.boolean  "new_event",      :default=>true
     t.boolean  "frequent_event", :default=>false
     t.boolean  "daily",          :default=>false
     t.boolean  "realtime",       :default=>false
-    t.string   "app_secret"
```

Comment:
You should try to put the app_secret back where it was and run `rake db:schema:load` so you have it after app_key in your database. That way when you run another migration for something else it does not get moved to this place.